tools/pipelines/publish-api-model-artifact.yml

```diff
@@ -66,7 +66,7 @@ variables:
   - name: shouldRetainGuardianAssets
     value: ${{ or(
       eq(parameters.guardianAssetRetentionOverride, 'force'),
-      and(eq(parameters.guardianAssetRetentionOverride, 'default'))
+      eq(parameters.guardianAssetRetentionOverride, 'default')
```

Comment:
It occurs to me that I don't actually know what these guardian assets are. Under what circumstances are we supposed to retain them?

If we really want to retain them by default, then I don't think we need a `force` option.